db/seeds.rb

```diff
@@ -3,6 +3,92 @@
 
 exit if Rails.env.test?
 FactoryBot.create :user, first_name: 'David', last_name: 'Faulkenberry', staff: true, spire: '12345678@umass.edu', email: 'dfaulken@umass.edu', admin: true
+user = FactoryBot.create :user, first_name: 'Justin', last_name: 'Forgue', spire: '13579246@umass.edu', email: 'jforgue@umass.edu'
```

Comment:
This is a public repository and I'm willing to bet that that's your actual spire ID.

Reply:
That is not my spire ID, I made that one up.

Reply:
Oh I thought you were being dumb my bad

Reply:
Apparently you also think he's been around a while... not even *your* SPIRE starts with a 1. 😛

Reply:
🖕